src/styles/_catalog.scss

```
@@ -14,9 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: $duration;
```

```suggestion
    transition: color $duration;
```
You should specify for which props you're applying transition